src/components/pretty-links-context.tsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react'
 
-export const PrettyLinksContext = React.createContext<
-  Record<string, { alias: string }>
->({})
+export type PrettyLinksContextProps =
+  | Record<string, { alias: string }>
+  | undefined
```

Comment:
Why do we need undefined? Doesn't `{}` cut it already?

Reply:
nice, I thought `Record<string, { alias: string }>` might not support that.